README.md

```diff
@@ -36,6 +36,23 @@ $ gpg-group-chat --server
 $ gpg-group-chat --client
 ```
 
+## Basic usage flow
+
+1. Server starts with gpg-group-chat --server at 192.168.0.1
+2. The client1 starts with gpg-group-chat --client --server-ip=192.168.0.1 --public\_key=public\_key1
+3. The client1 connect to the server and the server stores the public key for client1
+4. The server send the current list of keys for client1 (contains public\_key1)
+5. The client2 starts with gpg-group-chat --client --server-ip=192.168.0.1 --public\_key=public\_key2
+6. The client2 connect to the server and the server stores the public key for client2
+7. The server send the current list of keys for client1 and client2 (contains public\_key1 and publick\_key2)
```

Comment:
s/(contains public\_key1 and publick\_key2)/(contains public key1 and publick key2)